### ISSUE TITLE
Empty line in stderr when using --input-dir

### ISSUE THREAD

**Issue by CheyiLin** [hairyhenderson/gomplate#1057]
Refer to #996 

Similar issue occurred when using `--input-dir`:

```bash
# v3.6.0
$ asdf global gomplate v3.6.0
$ gomplate -v
gomplate version 3.6.0
$ gomplate --input-dir=templates
$

# v3.7.0
$ asdf global gomplate v3.7.0
$ gomplate -v
gomplate version 3.7.0
$ gomplate --input-dir=templates

$ gomplate --input-dir=templates 2>/dev/null
$
```

**Comment by hairyhenderson** [comment]
@CheyiLin Thanks for logging this. Just to be clear, I think this happens in v3.9.0 as well, right?

I believe I already have a fix for this, as part of #1050, though I was planning on moving it out. I thought I'd done that before shipping v3.9.0, but clearly not 😅

**Comment by CheyiLin** [comment]
@hairyhenderson Yes it happens in v3.7.0, v3.8.0 and v3.9.0 as well.
